polars/polars-lazy/polars-plan/src/logical_plan/pyarrow.rs

```diff
@@ -24,9 +44,9 @@ pub(super) fn predicate_to_pa(predicate: Node, expr_arena: &Arena<AExpr>) -> Opt
                 AnyValue::Boolean(val) => {
                     // python bools are capitalized
                     if val {
-                        Some("True".to_string())
+                        Some("pa.compute.scalar(True)".to_string())
```

Comment:
This change was required as returning `True`/ `False` was breaking currently as it gets evaluated to a bool. pyarrow requires an expression, not a bool